lib/platform/github/index.ts

```diff
@@ -219,13 +214,6 @@ export async function initRepo({
     if (!repo.defaultBranchRef?.name) {
       throw new Error(REPOSITORY_EMPTY);
     }
-    // istanbul ignore if
-    if (repo.isFork && !includeForks) {
-      const renovateConfig = await getJsonFile(defaultConfigFile);
-      if (!renovateConfig?.includeForks) {
-        throw new Error(REPOSITORY_FORKED);
-      }
-    }
```

Comment:
Are we okay for enabling this for other platforms too?

Reply:
sure. is it missing on one platform?

Reply:
Looks like it's present only for Github

Reply:
It would kind of be a backwards-incompatible "fix". But we do set `includeForks` to `false` by default so that should be the default behavior

Reply:
I think this is a small acceptable breaking fix, as our intend is `includeForks=false` by default (the docu says this too).